src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -71,12 +74,19 @@ const TreeItem: React.FC<TreeItemProps> = props => {
 
   const [isTitleHovered, setTitleHoverState] = useState(false);
 
+  const getGitTopLevel = async (local: string) => {
+    return promiseFromIpcRenderer('git.getGitTopLevel', 'git.getGitTopLevel.result', local);
+  };
+
   const fileOrFolderContainedInFilter = useAppSelector(state => state.main.resourceFilter.fileOrFolderContainedIn);
   const fileMap = useAppSelector(state => state.main.fileMap);
   const osPlatform = useAppSelector(state => state.config.osPlatform);
   const selectedPath = useAppSelector(state => state.main.selectedPath);
   const resourceMap = useAppSelector(state => state.main.resourceMap);
   const helmValuesMap = useAppSelector(state => state.main.helmValuesMap);
+  const git = useAppSelector(state => state.git);
+  const selectedProjectRootFolder = useAppSelector(state => state.config.selectedProjectRootFolder);
+  const remotePath = async () => (selectedProjectRootFolder ? getGitTopLevel(selectedProjectRootFolder) : null);
```

Comment:
Can we maybe switch it to a `useMemo` and use promiseFromIpcRenderer directly inside? In this way, we don't have to wait for creating the remote path ( when clicking on the open on GitHub option ), and only after that open the link